py/src/ai/h2o/sparkling/H2OContext.py

```diff
@@ -102,7 +102,6 @@ def _get_first(rdd):
 
 class H2OContext(object):
 
-    is_initialized = False
```

Comment:
We can't have state in Python client -> we need to move this state to scala part in order to allow the re connection.